### ISSUE TITLE
[bitnami/rabbitmq] Removal of curl in Dockerfile of container rabbitmq version 3.11.16-2

### ISSUE THREAD

**Issue by ysma500** [bitnami/containers#35694]
### Name and Version

bitnami/rabbitmq:3.11.16-debian-11-r5

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. Run container with community-plugins to download
2. Watch the container crash since it can't download plugins using curl, which is now absent

### What is the expected behavior?

Download the community plugin and start up.

### What do you see instead?

/opt/bitnami/scripts/librabbitmq.sh: line 514: curl: command not found

### Additional information

This is where curl is needed on startup: https://github.com/bitnami/containers/blob/main/bitnami/rabbitmq/3.11/debian-11/rootfs/opt/bitnami/scripts/librabbitmq.sh#L514 (same on 3.9 and 3.10)

This is a problem since "rabbitmq:3.11.16-debian-11-r5" but is also present on "rabbitmq:3.11.16-debian-11-r6".
Here is the problematic commit: (On the added L38) https://github.com/bitnami/containers/commit/a618a6d5acb2567bf5296b3f9a81f4dadd56d78a

Impacts all the deployments using the latest version of the bitnami/rabbitmq helm chart using community-plugins.

**Comment by ysma500** [comment]
Here's a [Pull Request](https://github.com/bitnami/containers/pull/35696) implementing the suggested changes.

**Comment by javsalgar** [comment]
Thank you so much for the PR! The team will take a look

**Comment by maximeroma** [comment]
3.9 and 3.9.29 are impacted too

**Comment by ysma500** [comment]
> 3.9 and 3.9.29 are impacted too

They are addressed in my suggested PR 👍

**Comment by carrodher** [comment]
Thanks for reporting the issue. After re-evaluating the change, we reverted the changes and `curl` is now re-added in our internal test & release pipeline which is in charge of releasing new versions of the container image as well as updating the source code present in this repo.

For instance, in https://github.com/bitnami/containers/pull/35981 or https://github.com/bitnami/containers/pull/35980 the specific line removing `curl` was removed.
